### DISCUSSION TITLE
The school you selected does not appear to have a campus location in your country

### DISCUSSION THREAD

**Discussion by bagheriali2001** [community/community#20819] [GitHub Education] [answered]
Hi,
I am trying to submit for GitHub Student Developer Pack, but when I upload a picture of my enrollment letter and run the preoccess my application it says 

> The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote.

The school I choose is in my country. I am currently connecting without any VPN, and I also updated my country in my GitHub profile (before this I didn't fill out the country field), on my previous attempts I got rejected because I didn't get a dated prof or it was not in English. When I finally get an English dated prof I get this error. I remember previously, I could enroll even with an issue but now I can't and I only have reprocess option. I have tried this with my personal and school email, but the results are the same. Is there any way to submit with this issue or submit without pre-check?
If anyone has had this issue before, how did you solve this?

**Comment by D7EAD** [comment]
Hi!

I would contact Github Education directly to resolve this issue. You can do so [here](https://support.github.com/contact/education).

**Reply by bagheriali2001** [reply]
Thanks a lot, I have submitted a ticket but didn't get any response in a day. How much is the usual wait time until they respond?

**Reply by wahab890** [reply]
Any update?

**Comment by ettaboyle** [comment, marked as answer]
Hi @bagheriali2001 👋🏻  If the education team was unable to verify your student status based on the information in your application you can fine more information in GitHub Help: [Why was my application for a student developer pack denied](https://help.github.com/articles/why-was-my-application-for-a-student-developer-pack-denied)

You are welcome to reapply with supplementary proof of your current student status at:
https://education.github.com/discount_requests/student_applicationhttps://education.github.com/pack

If you select a non-academic email or an untrusted academic email, you will be prompted to also upload additional official, dated proof of current school enrollment, such as your student ID.

Applications are generally processed within a few days, but may take longer in peak periods such as the start of a new semester.

**Reply by evilcat-x** [reply]
> Hi @bagheriali2001 👋🏻 If the education team was unable to verify your student status based on the information in your application you can fine more information in GitHub Help: [Why was my application for a student developer pack denied](https://help.github.com/articles/why-was-my-application-for-a-student-developer-pack-denied)
> 
> You are welcome to reapply with supplementary proof of your current student status at: https://education.github.com/discount_requests/student_applicationhttps://education.github.com/pack
> 
> If you select a non-academic email or an untrusted academic email, you will be prompted to also upload additional official, dated proof of current school enrollment, such as your student ID.
> 
> Applications are generally processed within a few days, but may take longer in peak periods such as the start of a new semester.

This issue is not resolved at all.
I can not even submit an application since the automatic checking system is saying I am not near a campus where it just don't know the School has a separate campus location.

And due to this restricting, when I open a ticket in Github Support, they automatically close my ticket and say I need to contact Github Education, and Github Education needs me to submit an application first.

So ppl who cannot submit an application due to your system error now have no way to even talk to a person to resolve the issue or report a bug.

**Reply by Kalyani02072003** [reply]
Hi, 
I faced the same issue as you. Exactly how you described it. I'd suggest you change your browser to firefox if you are using chrome currently. It helped me. I hope it helps you as well.

**Reply by chavels07** [reply]
Switching my browser from Chrome to Firefox really helps.

**Reply by kundanjan** [reply]
Switching browser not worked for me

**Reply by sryahel** [reply]
could you fix it?

**Reply by raihannisn** [reply]
works ?

**Reply by mhaemnn** [reply]
I have the same problem, and I have uploaded my student card but it still fails

**Comment by SigmaBM** [comment]
Same problem. Still not solved?

**Comment by barkatali-cpu** [comment]
The school I choose is in my country. I am currently connecting without any VPN, and I also updated my country in my GitHub profile (before this I didn't fill out the country field), on my previous attempts I got rejected because I didn't get a dated prof or it was not in English. When I finally get an English dated prof I get this error. I remember previously, I could enroll even with an issue but now I can't and I only have reprocess option. I have tried this with my personal and school email, but the results are the same. Is there any way to submit with this issue or submit without pre-check?
If anyone has had this issue before, how did you solve this?
Even I have used my University email, only my mail is not working other students studying in same semester has university mails are working.

**Comment by kevingnauh** [comment]
I am getting this error when trying to submit the application. The automated application review won't let me submit the app. I am not sure how to prove I am a distance learner. I submited support tickets but both were closed without a solution or explanation. Please advise. Thank you

**Comment by Dev-anshika98** [comment]
I got this error or problem when I submit my application in githup student developer pack
 Your application cannot be reviewed until you fix the following:
You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state 
Actually I think ,I filled wrong college name so please tell me how i resolve this problem

**Reply by ichigo-k** [reply]
Disable your VPN, then terminate the application and commence afresh.
